### PR TITLE
Interface list

### PR DESCRIPTION
**Pull request by therfert**
##### SUMMARY
 api_modify, api_info - support API paths:
  -  ``interface list``
  - ``interface list member``

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
api_info
api_modify

##### ADDITIONAL INFORMATION
N/A

### CODE REVIEW

File: plugins/module_utils/_api_data.py [hunk 1]
```diff
@@ -132,13 +132,23 @@ def join_path(path):
...
+        fully_understood=True,
         fields={
             'comment': KeyInfo(can_disable=True, remove_value=''),
             'exclude': KeyInfo(),
             'include': KeyInfo(),
             'name': KeyInfo(),
         },
     ),
+    ('interface', 'list', 'member'): APIData(
+        primary_keys=('interface','list', ),
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:144]: That should make the sanity tests happy:
```suggestion
        primary_keys=('interface', 'list', ),
```

Comment (therfert) [reply plugins/module_utils/_api_data.py]: @felixfontein yeah, I just noticed that sanity error and fixed in another commit

File: plugins/module_utils/_api_data.py [hunk 2]
```diff
@@ -132,13 +132,23 @@ def join_path(path):
...
+        fully_understood=True,
         fields={
             'comment': KeyInfo(can_disable=True, remove_value=''),
             'exclude': KeyInfo(),
             'include': KeyInfo(),
             'name': KeyInfo(),
         },
     ),
+    ('interface', 'list', 'member'): APIData(
+        primary_keys=('interface', 'list', ),
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:144]: While it makes no practical difference, I would change the order:
```suggestion
        primary_keys=('list', 'interface', ),
```

File: plugins/module_utils/_api_data.py [hunk 3]
```diff
@@ -132,13 +132,23 @@ def join_path(path):
...
+        fully_understood=True,
+        fields={
+            'comment': KeyInfo(can_disable=True, remove_value=''),
+            'interface': KeyInfo(),
+            'list': KeyInfo(),
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:149]: At least on RouterOS 6, there's also a `disabled` key:
```suggestion
            'list': KeyInfo(),
            'disabled': KeyInfo(default=False),
```

File: plugins/module_utils/_api_data.py (hunk 2 above)
Comment (therfert) [reply plugins/module_utils/_api_data.py:144]: I just used alphabetical order, but as you wish :)

File: plugins/module_utils/_api_data.py (hunk 3 above)
Comment (therfert) [reply plugins/module_utils/_api_data.py:149]: oh, right. I didn't test it on ROS6. Will add it

Comment (therfert) [reply plugins/module_utils/_api_data.py:149]: and actually it is in ROS7 as well.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@therfert thanks a lot for your contribution!

**Pull request merged**